R/table_participants_by_site.R

```diff
@@ -0,0 +1,60 @@
+#' Table of Participants Based on Site Selection
+#'
+#' @inheritParams shared-params
+#'
+#' @export
+
+table_participants_by_site <- function(
+    dfAnalyticsInput,
+    rctv_strSite) {
```

Comment:
```suggestion
    strSite) {
```
We pass this in static, so this isn't the `rctv_` version anymore. Also add `strSite` to `shared-params.